zowe/api-layer
# Service configuration validation

Related to https://github.com/zowe/api-layer/issues/570 . Add validator in the Plain Java enabler for the mandatory and non-mandatory parameters of a service configuration.

Validation provided:
* Check if a property is missing or if its value is not present
* Check, in case of wildcard, if the system property was set
